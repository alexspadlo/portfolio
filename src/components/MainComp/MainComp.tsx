import React, { FC } from 'react'
import WorkSection from '@components/WorkSection'
import StoreSection from '@components/StoreSection'
import HeroBanner from '@components/HeroBanner'
import s from './MainComp.scss'

interface MainCompProps {
    handlePopup: () => void
}

const MainComp: FC<MainCompProps> = ({ handlePopup }) => {
    return (
        <section className={`${s.txtcolWhite} h-100`}>
            <HeroBanner />
            <WorkSection handlePopup={handlePopup} />
            <StoreSection />
        </section>

    )
}

export default MainComp