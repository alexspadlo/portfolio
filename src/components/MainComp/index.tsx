export { default } from './MainComp'
export * from './MainComp'