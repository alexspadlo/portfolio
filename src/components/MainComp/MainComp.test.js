import React from 'react'
import renderer from 'react-test-renderer'
import MainComp from './MainComp'

describe('Test on <MainComp /> component', () => {
    test('Should render without the props', () => {
        const componentNoprops = renderer.create(<MainComp />)
        const treeNoprops = componentNoprops.toJSON()
        expect(treeNoprops).toMatchSnapshot()
    })
})
