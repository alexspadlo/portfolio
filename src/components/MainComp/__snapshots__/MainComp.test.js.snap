// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <MainComp /> component Should render without the props 1`] = `
<section
  className="txtcolWhite h-100"
>
  <div
    className="container-fluid my-lg-5 my-0 py-4"
  >
    <div
      className="row py-lg-5 py-0"
    >
      <div
        className="col-12 order-lg-1 order-2 col-lg-4 px-3"
      >
        <section
          className="w-100 text-right"
        >
          <p
            className="letterStyle"
          >
            PROGRAMMING
          </p>
          <p
            className="typeStyle"
          >
            JAVASCRIPT REACT  —  TYPESCRIPT -  MACHINE LEARNING  —  PYTHON
          </p>
          <p
            className="fontThin"
          >
            I'm a web developer hero based in London, with over 4 years experience in programming on multiple languages, creating useful functionality for users and better web experience.
          </p>
        </section>
      </div>
      <div
        className="col-12 order-lg-2 order-1 col-lg-4 pb-4 pb-lg-0"
      >
        <img
          alt="test-file-stub"
          className="img-fluid w-100"
          src="test-file-stub"
        />
      </div>
      <div
        className="col-12 order-lg-3 order-3 col-lg-4 px-3"
      >
        <section
          className="customHei d-lg-block d-none"
        >
           
        </section>
        <section
          className="w-100"
        >
          <p
            className="letterStyle"
          >
            DESIGN
          </p>
          <p
            className="typeStyle"
          >
            UI WEB DESIGN  —  ARTWORKS  —  COMMERCIAL
          </p>
          <p
            className="fontThin"
          >
            Designing it's my second superpower! From Web UI, banners, posters to artworks and small logos. And BTW, I can Design and Develop at the same time! How cool is that?! ;)
          </p>
        </section>
      </div>
    </div>
  </div>
  <div
    className="container-fluid my-5 p-0"
  >
    <div
      className="row"
    >
      <div
        className="col-12 col-lg-6 p-0"
      >
        <section
          className="codingStyle"
        >
          <p
            className="h-lg-2 h3 mb-3 fontBold"
          >
            Web/App UI Design & Specification
          </p>
          <p
            className="fontThin mb-4"
          >
            The challenge of developing these projects and all the knowledge I got from the work is what I like most about working as a UI Designer/Developer. Aligning business expectations with user-centered design to create better experiences, achieve goals and conversions through user testing, ideation, wireframes, prototyping, and layout design.
          </p>
          <section
            className="w-100"
          >
            <span
              aria-hidden="true"
              className="d-block d-md-inline-block   textLink"
              onClick={[Function]}
              role="button"
            >
              Take a look
            </span>
          </section>
        </section>
      </div>
      <div
        className="col-12 col-lg-6 p-0"
      >
        <section
          className="commonBg w-100"
        />
      </div>
    </div>
    <div
      className="row mt-5"
    >
      <div
        className="col-12 col-lg-6 p-0"
      >
        <section
          className="commonBg designBack w-100"
        />
      </div>
      <div
        className="col-12 col-lg-6 text-right p-0"
      >
        <section
          className="codingStyle"
        >
          <p
            className="h-lg-2 h3 mb-3 fontBold"
          >
            Commercial & Art Design
          </p>
          <p
            className="fontThin mb-4"
          >
            From the point of origin comes the idea, a scribble on paper, digitization and the end product. Each process is pure magic and it feels like I'm not at work, but in an intellectual therapy to develop creativity and to be happy with myself. Main goal as being a Designer is to see that full joy on people faces, enjoying my creativity.
          </p>
          <section
            className="w-100"
          >
            <span
              aria-hidden="true"
              className="d-block d-md-inline-block   textLink"
              onClick={[Function]}
              role="button"
            >
              Take a look
            </span>
          </section>
        </section>
      </div>
    </div>
  </div>
  <div
    className="container-fluid my-5 py-4"
  >
    <div
      className="row py-5"
    >
      <div
        className="col-12 col-lg-7 m-auto"
      >
        <section
          className="w-100 text-center"
        >
          <p
            className="fontBold h2"
          >
            Artworks Showcase
          </p>
          <p
            className="fontThin"
          >
            Art is my safe haven, where I flee in the sad days and also in the happy days to expose my inspirations.
          </p>
          <section
            className="w-100"
          >
            <span
              aria-hidden="true"
              className="d-block d-md-inline-block   textLink"
              onClick={[Function]}
              role="button"
            >
              View Artworks
            </span>
          </section>
        </section>
      </div>
    </div>
    <section
      className="w-100 position-relative"
    >
      <span
        className="topBorder"
      >
         
      </span>
      <div
        className="row"
      >
        <div
          className="col-sm text-center"
        >
          <section
            className="bg-white h-100 pointer"
            onClick={[Function]}
          >
            <img
              alt="test-file-stub"
              className="imgClass p-2"
              src="test-file-stub"
            />
          </section>
        </div>
        <div
          className="col-sm text-center"
        >
          <section
            className="bg-white h-100 pointer"
            onClick={[Function]}
          >
            <img
              alt="test-file-stub"
              className="imgClass p-2"
              src="test-file-stub"
            />
          </section>
        </div>
        <div
          className="col-sm text-center"
        >
          <section
            className="bg-white h-100 pointer"
            onClick={[Function]}
          >
            <img
              alt="test-file-stub"
              className="imgClass p-2"
              src="test-file-stub"
            />
          </section>
        </div>
        <div
          className="col-sm text-center"
        >
          <section
            className="bg-white h-100 pointer"
            onClick={[Function]}
          >
            <img
              alt="test-file-stub"
              className="imgClass p-2"
              src="test-file-stub"
            />
          </section>
        </div>
        <div
          className="col-sm text-center"
        >
          <section
            className="bg-white h-100 pointer"
            onClick={[Function]}
          >
            <img
              alt="test-file-stub"
              className="imgClass p-2"
              src="test-file-stub"
            />
          </section>
        </div>
      </div>
      <span
        className="bottomBorder"
      >
         
      </span>
    </section>
  </div>
  <div
    className="container-fluid my-5 py-4"
  >
    <div
      className="row py-5"
    >
      <div
        className="col-12 col-lg-7 m-auto"
      >
        <section
          className="boxStyle"
        >
          <div
            className="row m-0"
          >
            <div
              className="col-12 col-lg-6 p-0"
            >
              <img
                alt="test-file-stub"
                className="img-fluid w-100"
                src="test-file-stub"
              />
            </div>
            <div
              className="col-12 col-lg-6 p-0"
            >
              <section
                className="w-100 text-center py-5 py-lg-4 px-4 h-100 position-relative"
              >
                <section
                  className="centerAlign position-relative position-lg-absolute"
                >
                  <p
                    className="fontBold h4"
                  >
                    Support my Art
                  </p>
                  <p
                    className="fontThin"
                  >
                    Like my Artworks? Feel free to check my store, where you can find tons of products with my art printed on. By purchase them, you supporting my creativity (before AI will take over) and give me motivation for more :)
                  </p>
                  <section
                    className="w-100"
                  >
                    <span
                      aria-hidden="true"
                      className="d-block d-md-inline-block   textLink"
                      onClick={[Function]}
                      role="button"
                    >
                      View shop
                    </span>
                  </section>
                </section>
              </section>
            </div>
          </div>
        </section>
      </div>
    </div>
  </div>
</section>
`;
